src/shiver/presenters/histogram.py

```diff
@@ -55,3 +60,62 @@ def delete_workspace(self, name):
     def rename_workspace(self, old_name, new_name):
         """Called by the view to rename a workspace"""
         self.model.rename(old_name, new_name)
+
+    def create_corrections_tab(self, name):
+        """Create a corrections tab"""
+        tab_name = f"Corrections - {name}"
+        tab_widget = self.view.parent().parent()
+
+        # check if tab already exists
+        tab_idx = tab_widget.indexOf(tab_widget.findChild(QWidget, tab_name))
+        if tab_idx != -1:
+            tab_widget.setCurrentIndex(tab_idx)
+        else:
+            # create a new tab
+            corrections_tab_view = Corrections(parent=self.view, name=name)
+            corrections_tab_view.setObjectName(tab_name)
+            # create a new model
+            corrections_tab_model = CorrectionsModel()
+
+            # inline functions
+            def _show_error(msg):
+                """Show an error message"""
+                corrections_tab_view.show_error_message(msg)
+
+            def _help():
+                """Show the help for the corrections tab"""
+                webbrowser.open("https://neutrons.github.io/Shiver/")
+
+            def _apply():
+                """Apply the corrections"""
+                corrections_tab_model.apply(
+                    name,
+                    corrections_tab_view.detailed_balance.isChecked(),
+                    corrections_tab_view.hyspec_polarizer_transmission.isChecked(),
+                    corrections_tab_view.temperature.text(),
+                )
+                # clean up
+                tab_widget.setCurrentWidget(self._view)
+                corrections_tab_view.deleteLater()
+
+            def _cancel():
+                """Cancel the corrections tab"""
+                tab_widget.setCurrentWidget(self._view)
+                corrections_tab_view.deleteLater()
+
+            # populate initial values
+            has_detailed_balance, temperature = corrections_tab_model.has_apply_detailed_balance(name)
+            has_scattered_transmission_correction = corrections_tab_model.has_scattered_transmission_correction(name)
+            if has_detailed_balance:
```

Comment:
Since we don't undo the corrections, these options should be disabled (checkboxes and temperature string)